src/cpp/src/tokenizer.cpp

```diff
@@ -707,6 +742,21 @@ void Tokenizer::set_chat_template(const std::string& chat_template) {
     m_pimpl->set_chat_template(chat_template);
 }
 
+Vocab Tokenizer::get_vocab() const {
```

Comment:
Should `detokenizer` be added to the vars and this function name so it would be clear that it comes not from `tokenizer`?

Reply:
The method mirrors the name from the Huggingface tokenizer object.